src/kernel/src/machine/arm/gicv2/gicd.rs

```diff
@@ -0,0 +1,145 @@
+/// GICv2 Distributor Interface
```

Comment:
Can you either expand the acronym and/or point to external (and stable) documentation of what this is?